app/controllers/verify/confirmations_controller.rb

```diff
@@ -11,10 +11,6 @@ def index
       finish_proofing_success
     end
 
-    def continue
-      redirect_to after_sign_in_path_for(current_user)
```

Comment:
Yes, we are sending them to `sign_up_recovery_code_path` via that confirmation modal ( https://github.com/18F/identity-idp/blob/master/app/views/sign_up/recovery_codes/_confirmation_modal.html.slim) which goes to `update` in the `SignUp::RecoveryCodesController` 

https://github.com/18F/identity-idp/blob/c2e1184f30666561715add5fc78447dbd666b4c3/app/controllers/sign_up/recovery_codes_controller.rb#L14

This controller action is old / unused currently